apps/addons/models.py

```diff
@@ -624,12 +624,15 @@ def share_url(self):
 
     @property
     def automated_signing(self):
-        # We allow automated signing for add-ons which are not listed, and
-        # have not asked for side-loading privileges (full review).
+        # We allow automated signing for add-ons which are not listed.
         # Beta versions are a special case for listed add-ons, and are dealt
         # with on a file-by-file basis.
-        return not (self.is_listed or
-                    self.status in (amo.STATUS_NOMINATED, amo.STATUS_PUBLIC))
```

Comment:
According to https://bugzilla.mozilla.org/show_bug.cgi?id=1229197#c1 we want to just sign unlisted, whether side-load or not.